bluej/package/build.xml

```diff
@@ -223,9 +223,11 @@
             <bundledocument extensions="bjar"
                             name="BlueJ archive"
                             role="Editor"/>
-            <runtime dir="${mac_bundled_jdk_path}">
-            </runtime>
+            <runtime dir="${mac_bundled_jdk_path}"/>
             <arch name="x86_64"/>
+            <environment name="JAVA_HOME" value="$APP_ROOT/Contents/PlugIns/x64/Contents/Home">
+
+            </environment>
```

Comment:
Just for clarity: I think we can collapse the tag <environment> like it was done for <runtime> just above